### PR TITLE
DCS-1302: Update project dependencies to fix failing OWASP dependency checks

### PR DESCRIPTION
**Pull request by bell-pa**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@3.11
+  hmpps: ministryofjustice/hmpps@3.11.11
```
Comment (petergphillips) [inline .circleci/config.yml:4]: not normally a good idea to pin to specific orb version - better to let it float at `3.11`

**Review by louisenorris — DISMISSED** [review]
LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (bell-pa) [reply .circleci/config.yml:4]: Damn.  I think you said that last time and I forgot. Again...

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**